app/controllers/merchant/items_controller.rb

```diff
@@ -3,4 +3,10 @@ class Merchant::ItemsController < ApplicationController
   def index
     @merchant = Merchant.find(current_user[:merchant_id])
   end
+
+  def destroy
+    Item.destroy(params[:id])
+    flash[:error] = "Item Deleted"
```

Comment:
Used error flash to have it be red

Reply:
![](https://media1.giphy.com/media/3o7qE0gOGwzPbH81Qk/giphy.gif?cid=5a38a5a23de21b19e107bc9464e4b9e437360d3030afda71&rid=giphy.gif)